ignite/distributed/comp_models/native.py

```diff
@@ -215,12 +215,6 @@ def setup_env_vars(self, rank: Optional[int] = None, world_size: Optional[int] =
             all_env_vars_defined = [k in os.environ for k in env_vars]
```

Comment:
L214-215 should be moved in the `else` statement L22

Reply:
L22 ?